mex/extractors/biospecimen/transform.py

```diff
@@ -95,10 +97,10 @@ def transform_biospecimen_resource_to_mex_resource(  # noqa: PLR0913
 
         contact: list[Identifier] = []
         for kontakt in resource.kontakt:
-            if k := person_stable_target_id_by_email.get(kontakt):  # noqa: SIM114
-                contact.append(k)
-            elif k := unit_stable_target_ids_by_synonym.get(kontakt):
+            if k := person_stable_target_id_by_email.get(kontakt):
                 contact.append(k)
+            elif k := unit_stable_target_ids_by_synonym.get(kontakt, []):  # type: ignore[assignment]
+                contact.extend(k)
```

Comment:
if we use a new variable name for the units, then we don't need the type-ignore.
also, with the walrus, you don't need the default value of `[]`

```suggestion
            if person := person_stable_target_id_by_email.get(kontakt):
                contact.append(person)
            elif unit := unit_stable_target_ids_by_synonym.get(kontakt):
                contact.extend(unit)
```